docs/gnosis_safe_app_support.md

```diff
@@ -1,6 +1,6 @@
 ## Running as a Gnosis Safe App
 
-It is possible to run the ENS app as a [Gnosis Safe App](https://docs.gnosis.io/safe/docs/sdks_safe_apps/). This will allow it to directly use the ENS app from withing the [Gnosis Safe web interface](https://app.gnosis-safe.io).
+It is possible to run the ENS app as a [Gnosis Safe App](https://docs.gnosis.io/safe/docs/sdks_safe_apps/). This will allow it to directly use the ENS app from within the [Gnosis Safe web interface](https://app.gnosis-safe.io).
```

Comment:
```suggestion
It is possible to run the ENS app like a [Gnosis Safe App](https://docs.gnosis.io/safe/docs/sdks_safe_apps/). This will allow it to directly use the ENS app from within the [Gnosis Safe web interface](https://app.gnosis-safe.io).
```